_posts/platform/app/2000-01-01-procfile.md

```diff
@@ -36,9 +36,12 @@ The standard syntax is:
 ```yaml
 <container type>: <command>
 
-container type: should contain letters and digits only
-command: any command line which can includes environment variables
+# where container type should contain letters and digits only
+# and command can be any command line which can includes environment variables
 ```
+As a note,`container type` are unique in the `Procfile`. So, you cannot have, for example, two `web` `container type`.
+If you have, only the second one will be executed, as if the first did not exist.
```

Comment:
I wouldn't explain the behaviour in case there is two container type described in the Procfile. We actually don't want anybody to have the same name for multiple container types. Hence we should just say: "don't do that or you may have unexpected behaviour". Don't you think?

Reply:
There is a ticket to add a fatal error in the procfile has two targets with the same name, but it is done yet. I will edit the docs to include the behavior once the ticket is done

Reply:
Can you add a link to this ticket in this PR so that it is reflected in GitHub UI?

Reply:
You talk about an issue but I don't see it mentioned in the PR. Can you add a reference to this issue?

Reply:
You didn't addressed my previous message in this thread.

Reply:
https://github.com/Scalingo/api/issues/1747